app/models/audio_file.rb

```diff
@@ -26,6 +26,7 @@ class AudioFile < BaseModel
   alias_attribute :duration, :length
 
   mount_uploader :file, AudioFileUploader, mount_on: :filename
+  skip_callback :commit, :after, :remove_file! # don't remove s3 file
```

Comment:
I think it was [this post](http://stackoverflow.com/questions/17302678/i-would-like-to-stop-carrierwave-deleting-a-file) that led me to believe that only applies to updates.  Will have to double-check that - never actually tried that config.